makefile

```diff
@@ -0,0 +1,62 @@
+SERVICES = rust-actix
```

Comment:
Do we need to define the other services/applications here as well?

Reply:
yes